src/test/run-pass/issue-3121.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// pretty-expanded FIXME #23616
+// no-pretty-expanded FIXME #23616 assert_receiver_is_clone
```

Comment:
Nah yeah it's fine to just remove everywhere, right now that phase of testing is opt-in